.github/workflows/execution.yml

```
@@ -47,16 +47,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code into the Go module directory
-        uses: actions/checkout@v3
-      - name: Set up Go 1.23
-        uses: actions/setup-go@v4
+        uses: actions/checkout@v4
+      - name: Set up Go ${{ matrix.go }}
```

actionlint (1.7.7)

property "go" is not defined in object type {}

(expression)